symbolic-il2cpp/src/usym.rs

```diff
@@ -474,7 +568,8 @@ mod tests {
 
         let mut last_address = usyms.records[0].address;
         for i in 1..usyms.header.record_count as usize {
-            assert!(usyms.records[i].address > last_address);
+            // The addresses should be weakly monotonic
+            assert!(usyms.records[i].address >= last_address);
```

Comment:
added this in while investigating the usym with @loewenheim, we'd discovered that these should actually be `>=` instead of strictly `>`.